src/sentry/snuba/discover.py

```diff
@@ -458,7 +459,10 @@ def timeseries_query(selected_columns, query, params, rollup, referrer=None):
                 # Check that the column is a list with 3 items, and the alias in the third item is an equation
                 if isinstance(column, list)
                 and len(column) == 3
-                and column[-1].startswith("equation[")
+                and (
+                    column[-1].startswith("equation[")
+                    or column[-1] == PROJECT_THRESHOLD_CONFIG_ALIAS
```

Comment:
What if there are multiple projects configured with custom thresholds? This would result in an additional group by on the `PROJECT_THRESHOLD_CONFIG_ALIAS` right which means a single timestamp can have more than 1 result in the query.